DataSource/DataSourceBuilder.php

```diff
@@ -168,6 +169,9 @@ private function getDataTypeByName($type)
             'entity' => function () {
                 return new EntityDataType();
             },
+            'percent' => function () {
```

Comment:
Also not nice... i would like to investigate a better way of handling this at some point